cmd/fleet/serve.go

```diff
@@ -678,7 +678,7 @@ func runCrons(
 	ds fleet.Datastore,
 	task *async.Task,
 	logger kitlog.Logger,
-	config configpkg.FleetConfig,
+	config *configpkg.FleetConfig,
```

Comment:
Assuming the goroutines started here do not modify the provided config pointer arguments, it seems it would be safer to make a copy here (i.e. leave as it was).

Reply:
that's a good idea 👍